### PR TITLE
[14.0][MIG] Animal owner

### PR DESCRIPTION
**Pull request by bealdav**

### CODE REVIEW

**Comment by bealdav** [comment]
cc @max3903 

runbot is up but odoo logs unreachable

File: animal_owner/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Copyright (C) 2020 Open Source Integrators
```
Comment (NL66278) [inline animal_owner/__init__.py]: Remove copyright lines from init files.

File: animal_owner/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (NL66278) [inline animal_owner/__init__.py]: Make all license lines https;

**Review by max3903 — APPROVED** [review]

**Review by LESTRAT21 — APPROVED** [review]

**Comment by bealdav** [comment]
runbot sank one more time, all other process are OK
runboat won the race
@gurneyalex could you merge ?
cc @max3903 @sbidoul

**Comment by bealdav** [comment]
Probably the time to disconnect runbot will happen. 
I hope it can be shortly

**Pull request merged**